src/common/knexConf.js

```diff
@@ -0,0 +1,22 @@
+var db_conf = require('../../database.json');
+var env = process.env.NODE_ENV == 'test' ? 'development' : 'default';
```

Comment:
Use `const` or `let` instead of `var`  and use `===` instead of `==` according linter